### ISSUE TITLE
Libvirt instead of VirtualBox error

### ISSUE THREAD

**Issue by ghost** [stanma/vagrant#1]
Hi, I am using CentOS and during make vagrant up command I got next:
Bringing machine 'default' up with 'libvirt' provider...
==> default: Box 'hashicorp/precise32' could not be found. Attempting to find and install...
    default: Box Provider: libvirt
    default: Box Version: >= 0
==> default: Adding box 'hashicorp/precise32' (v0) for provider: libvirt
    default: Downloading: http://files.vagrantup.com/precise32.box
The box you attempted to add doesn't match the provider you specified.

Provider expected: libvirt
Provider of box: virtualbox

Seems like Vagrantfile support only VirtualBox images? Is it possible to add both configurations?

**Comment by stanma** [comment]
yes, you're right, right now it is only for VirtualBox images. By "both" what do you mean? Other providers like VMWare?

**Comment by ghost** [comment]
I mean support libvirt/KVM and of course other providers, if you wish)

**Comment by stanma** [comment]
I see. It was planned. I'll add it.

**Comment by ghost** [comment]
That would be great!
